scripts/variantstore/wdl/GvsPrepareBulkImport.wdl

```diff
@@ -0,0 +1,82 @@
+version 1.0
+
+import "GvsUtils.wdl" as Utils
+
+workflow GvsPrepareBulkImport {
+    input {
+        String project_id
+        String workspace_name
+        String workspace_bucket
+        String samples_table_name
+        String sample_id_column_name
+        String vcf_files_column_name
+        String vcf_index_files_column_name
+    }
+
+    call GenerateFOFNsFromDataTables {
+        input:
+            project_id = project_id,
+            workspace_name = workspace_name,
+            workspace_bucket  = workspace_bucket,
+            samples_table_name = samples_table_name,
+            sample_id_column_name = sample_id_column_name,
+            vcf_files_column_name = vcf_files_column_name,
+            vcf_index_files_column_name = vcf_index_files_column_name
+    }
+
+    output {
+        File sampleFOFN = GenerateFOFNsFromDataTables.sampleFOFN
+        File vcfFOFN = GenerateFOFNsFromDataTables.vcfFOFN
+        File vcfIndexFOFN = GenerateFOFNsFromDataTables.vcfIndexFOFN
+        File errorRows = GenerateFOFNsFromDataTables.errors
+    }
+}
+
+# This copies the entirety of the data tables over.  We'll want a different task to make FOFNs for just a sample set
+task GenerateFOFNsFromDataTables {
+    input {
+        String project_id
+        String workspace_name
+        String workspace_bucket
+        String samples_table_name = "sample"
+        String sample_id_column_name = "sample_id"
+        String vcf_files_column_name = "hg38_reblocked_v2_vcf"
+        String vcf_index_files_column_name = "hg38_reblocked_v2_vcf_index"
+    }
+
+    ## TODO currently we can only take in samples, but we need to be able to take in sample_sets
```

Comment:
should this be a separate issue or part of https://broadworkbench.atlassian.net/browse/VS-835?

Reply:
good idea!